src/config.ts

```diff
@@ -91,7 +91,10 @@ export class Config implements ConfigInterface {
   }
 
   private parseBuffer(buf: Buffer): void {
-    const lines = buf.toString().split('\n');
+    const lines = buf
+      .toString()
+      .split('\n')
+      .filter((v) => v);
```

Comment:
Why separate to 4 lines? The original row length is not exceeding the limit I think.

Reply:
this was fromatted by prettier,eslint forced this format

Reply:
OK